analyses/cnv-frequencies/01-cnv-frequencies.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+
+
+"""
+01-snv-frequencies.py
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Functions to create copy number variation (CNV) cancer type and study gene-level frequencies for OPenPedCan analyses modules 
+"""
+
+
+__author__ = ('Eric Wafula (wafulae@chop.edu)')
+__version__ = '1.0'
+__date__ = '12 July 2021'
+
+
+import os 
+import sys
+import csv
+import json
+import argparse
+import subprocess
+import numpy as np
+import pandas as pd
+from functools import reduce
+from collections import OrderedDict
+
+
+def read_parameters():
+     p = argparse.ArgumentParser(description=("The 01-snv-frequencies.py scripts creates copy number variation (CNV) cancer type and study gene-level alterations frequencies table for the OPenPedCan analyses modules."), formatter_class=argparse.RawTextHelpFormatter)
+     p.add_argument('HISTOLOGY_FILE', type=str, default=None, help="OPenPedCan histology file (histologies.tsv)\n\n")
+     p.add_argument('CNV_FILE', type=str, default=None, help="OPenPedCan CNV consensus file (consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz and consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz)\n\n")
+     p.add_argument('PRIMARY_TUMORS', type=str, default=None, help="OPenPedCan independent primary tumor samples file (independent-specimens.wgs.primary.eachcohort.tsv)\n\n")
+     p.add_argument('RELAPSE_TUMORS', type=str, default=None, help="OPenPedCan independent relapse tumor samples file (independent-specimens.wgs.relapse.eachcohort.tsv)\n\n")
+     p.add_argument('-v', '--version', action='version', version="classify_reads.py version {} ({})".format(__version__, __date__), help="Print the current 01-snv-frequencies.py version and exit\n\n")
```

Comment:
You will need to use the independent specimen files which contain wxs and panel data here now that TARGET is in the batch.

Reply:
Resolved in <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/52/commits/f18ca249601a355d2f721d307eaee38eb2782e60>.

<https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/f18ca249601a355d2f721d307eaee38eb2782e60/analyses/cnv-frequencies/run-cnv-frequencies-analysis.sh#L22-L26>

README.md is also updated accordingly.